storages/backends/s3.py

```diff
@@ -167,35 +215,24 @@ def closed(self):
 
     def _get_file(self):
         if self._file is None:
-            self._file = tempfile.SpooledTemporaryFile(
-                max_size=self._storage.max_memory_size,
-                suffix=".S3File",
-                dir=setting("FILE_UPLOAD_TEMP_DIR"),
-            )
             if "r" in self._mode:
                 self._is_dirty = False
                 params = _filter_download_params(
                     self._storage.get_object_parameters(self.name)
                 )
-                self.obj.download_fileobj(
-                    self._file, ExtraArgs=params, Config=self._storage.transfer_config
-                )
-                self._file.seek(0)
+                self._file = S3SeekableFile(self.obj, ExtraArgs=params)
```

Comment:
Please note that `Config=self._storage.transfer_config` was removed... I'm not sure if this is needed as the method changed from `download_fileobj` to `get` as well.